app/views/histories/index.html.erb

```diff
@@ -16,14 +20,21 @@
     <hr class="keyline">
     <div class="column copy">
```

Comment:
If we're introducing govuk-frontend's grid elsewhere in this PR, is there a reason not to do it here as well? (sorry, don't want to make more work for you unnecessarily)

Reply:
You've nerdsniped me into doing this.